packages/redux-dynostore-react-redux-subspace/src/subspaced.js

```diff
@@ -9,7 +9,11 @@
 import { subspaced } from 'react-redux-subspace'
 import { defaultStateHandler } from '@redux-dynostore/core'
 
-const subspacedEnhancer = ({ stateHandler, mapExtraState = () => null } = {}) => identifier => store => {
+const subspacedEnhancer = ({
+  stateHandler,
+  subspaceOptions,
+  mapExtraState = () => null,
+} = {}) => identifier => store => {
```

Comment:
Firstly thanks for taking the time to try and improve things.

As much as possible, the `subspaced` enhancer is supposed to be analogues with the [`subspaced`](https://ioof-holdings.github.io/redux-subspace/packages/react-redux-subspace/docs/api/subspaced.html) HOC itself.  With that in mind, I think having the API be just

```js
dynamic(
  'identifier',
  subspaced({ context }),
  attachReducer(reducer),
  runSaga(saga),
  { context },
)
```

fits better with that vision.  For the sake of future compatibility, I would implement this as

```suggestion
const subspacedEnhancer = ({
  stateHandler,
  mapExtraState = () => null,
  ...subspaceOptions
} = {}) => identifier => store => {
```

Reply:
Also, can you add [some docs](https://github.com/ioof-holdings/redux-dynostore/tree/master/packages/redux-dynostore-react-redux-subspace#options) around the usage of this.

Maybe just a line saying any additional options are passed through as options to the subspace library with a link to the [API docs](https://ioof-holdings.github.io/redux-subspace/packages/react-redux-subspace/docs/api/subspaced.html#arguments) there?